### PR TITLE
1.26.0

### PR DESCRIPTION
**Pull request by pq**
# 1.26.0

- new lint: `combinators_ordering`
- fixed `use_colored_box` and `use_decorated_box` to not over-report
  on containers without a child
- fixed false positive for `unnecessary_parenthesis` on a map-or-set
  literal at the start of an expression statement
- fixed false positive for `prefer_final_locals` reporting on fields
- fixed `unnecessary_overrides` to allow overrides on `@Protected`
  members
- fixed `unnecessary_parenthesis` false positives on map and set
  literal at the beginning of expressions
- fixed `avoid_multiple_declarations_per_line` false positive in
  `for` statements
- fixed `prefer_final_locals` false positive on declaration lists
  with at least one non-final variable
- fixed `use_build_context_synchronously` to handle `await`s in
  `if` conditions

---

/cc @srawlins @bwilkerson @a14n

### CODE REVIEW

**Review by bwilkerson — APPROVED** [review]

**Comment by pq** [comment]
Trying an SDK roll w/ this release here: https://dart-review.googlesource.com/c/sdk/+/250949

(For the curious, we'll want to keep an eye on potential new reports of `use_build_context_synchronously`.)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
+- fixed `use_colored_box` and `use_decorated_box` to not over-report
+  on containers without a child
 - fixed false positive for `unnecessary_parenthesis` on a map-or-set
   literal at the start of an expression statement
+- fixed false positive for `prefer_final_locals` reporting on fields
+- fixed `unnecessary_overrides` to allow overrides on `@Protected`
+  members
+- fixed `unnecessary_parenthesis` false positives on map and set
+  literal at the beginning of expressions
```
Comment (a14n) [inline CHANGELOG.md:11]: duplication with the second item?

Comment (pq) [reply CHANGELOG.md]: Ah, yes.  Good catch.  Thanks!

**Comment by pq** [comment]
(Added `combinators_ordering`; 🎉 @a14n!)

**Pull request merged**